src/custom-element.js

```diff
@@ -282,6 +284,9 @@ export function createAmpElementProto(win, name, implementationClass) {
     /** @private {string|null|undefined} */
     this.mediaQuery_;
 
+    /** @private {!SizeList|null|undefined} */
```

Comment:
no expert with closure type annotations, but can't it be just `{?SizeList|undefined}` ?